lib/signaling/v2/room.js

```diff
@@ -166,10 +180,25 @@ RoomV2.prototype._update = function _update(roomState) {
     this.recording.update(roomState.recording);
   }
 
-  if (roomState.published) {
+  if (roomState.published && roomState.published.revision > this._publishedRevision) {
+    this._publishedRevision = roomState.published.revision;
+    roomState.published.tracks.forEach(function(track) {
```

Comment:
@markandrus Shouldn't we also remove any tracks that have been unpublished?

Reply:
No, because `getStats` provides historical data, even after MediaStreamTracks have been removed. We should keep this set up to date in order to 1. filter, e.g., for this use case, but also 2. to be able to insert Track SIDs (in a future PR).